src/Controllers/MapController.ts

```diff
@@ -1419,14 +1428,22 @@ module StreamStats.Controllers {
                     },
                     onEachFeature: function (feature, layer) {
                         var siteNo = feature.properties['Code'];
-                        var SSgagepage = 'https://streamstatsags.cr.usgs.gov/gagepages/html/' + siteNo + '.htm';
+                        var urls = ['https://streamstatsags.cr.usgs.gov/NC_gagePages/Sta_' + siteNo + '_daily_discharge_percentiles_table_by-wateryears.txt',
+                        'https://streamstatsags.cr.usgs.gov/NC_gagePages/Sta_' + siteNo + '_daily_discharge_percentiles_table_by-day-month-seasonal.txt',
+                        'https://streamstatsags.cr.usgs.gov/IA_gagePages/' + siteNo + '_stats.pdf'];
+                        var text = ['Flow-Duration Statistics by Water Years:',
+                        'Flow-Duration Statistics by PoR, Calendar Day & Month, & Seasonal Periods:',
+                        'Stream Flow Statistics:'];
                         var NWISpage = 'http://nwis.waterdata.usgs.gov/nwis/inventory/?site_no=' + siteNo;
                         var gageButtonDiv = L.DomUtil.create('div', 'innerDiv');
+                        var gageButtonLoaderDiv = L.DomUtil.create('div', 'innerDiv');
 
                         gageButtonDiv.innerHTML = '<strong>Station ID: </strong>' + siteNo + '</br><strong>Station Name: </strong>' + feature.properties['Name'] + '</br><strong>Latitude: </strong>' + feature.geometry.coordinates[1] + '</br><strong>Longitude: </strong>' + feature.geometry.coordinates[0] + '</br><strong>Station Type</strong>: ' + feature.properties.StationType.name +
-                            '</br><strong>NWIS page: </strong><a href="' + NWISpage + ' "target="_blank">link</a></br><strong>StreamStats Gage page: </strong><a href="' + SSgagepage + '" target="_blank">link</a></br><strong>New StreamStats Gage Modal: </strong><a id="gagePageLink" class="' + siteNo + '">link</a><br>';
+                            '</br><strong>NWIS Page: </strong><a href="' + NWISpage + ' "target="_blank">link</a></br><strong>StreamStats Gage Modal: </strong><a id="gagePageLink" class="' + siteNo + '">link</a><br>';

```

Comment:
I would change "Modal" to "Page" because I think most people don't know what a modal is?